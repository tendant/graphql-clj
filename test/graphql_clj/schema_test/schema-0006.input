interface NamedEntity {
name: String
}

type Person implements NamedEntity {
name: String
age: Int
}

type Business implements NamedEntity {
name: String
employeeCount: Int
}