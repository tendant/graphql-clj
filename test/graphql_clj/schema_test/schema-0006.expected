{:interfaces
 {:NamedEntity {:name "NamedEntity", :fields {:name {:type String}}}},
 :objects
 {:Person
  {:implements [:NamedEntity],
   :fields {:name {:type String}, :age {:type Int}}},
  :Business
  {:implements [:NamedEntity],
   :fields {:name {:type String}, :employeeCount {:type Int}}}}}
